packages/docs/src/theme/DocBreadcrumbs/index.js

```diff
@@ -0,0 +1,97 @@
+/**
```

Comment:
can you add a comment with the URL and hash the file is from?

Reply:
@lorensr done :+1: